Explorer/Assets/DCL/PluginSystem/Global/Global Plugins Settings.asset

```diff
@@ -911,4 +911,4 @@ MonoBehaviour:
           m_SubObjectName: 
           m_SubObjectType: 
           m_SubObjectGUID: 
-          m_EditorAssetChanged: 0
+          m_EditorAssetChanged: 0
```

Comment:
can you revert this so we don't get conflicts? nothing changed :D

Reply:
Of course